src/main/scala/io/smartcat/twicas/pipeline/PipelineProcessor.scala

```diff
@@ -0,0 +1,32 @@
+package io.smartcat.twicas.pipeline
+
+import java.io._
+
+import org.apache.spark.sql.DataFrame
+
+
+class PipelineProcessor(pipelines: List[Pipeline]) extends Serializable {
+
+  def processAll(df: DataFrame): DataFrame = pipelines.foldLeft(df)((dff, pipeline) => pipeline.process(dff))
+
+}
+
+object PipelineProcessor extends Serializable {
+
+  def loadFromFile(filepath: String): PipelineProcessor = {
+    val loader = Thread.currentThread().getContextClassLoader
+    val objIn = new ObjectInputStream(new FileInputStream(filepath)) {
```

Comment:
Because of exception when deserializing list.
https://issues.scala-lang.org/browse/SI-9777